packages/runtime/container-runtime/src/test/dataStoreContext.spec.ts

```diff
@@ -21,18 +31,32 @@ import { MockFluidDataStoreRuntime } from "@fluidframework/test-runtime-utils";
 import { SummaryTracker, createRootSummarizerNodeWithGC } from "@fluidframework/runtime-utils";
 import { IsoBuffer, TelemetryNullLogger } from "@fluidframework/common-utils";
 import {
+    attributesBlobKey,
+    gcBlobKey,
     IFluidDataStoreAttributes,
     LocalFluidDataStoreContext,
     RemotedFluidDataStoreContext,
 } from "../dataStoreContext";
 import { ContainerRuntime } from "../containerRuntime";
 
+function getBlobFromSnapshot(blobKey: string, snapshotTree: ITree): IBlob {
```

Comment:
It feels like we do the same in shipping code when working with storage, no? I.e. is there any chance of reusing existing code?

Reply:
I did not find anything similar. The storage APIs usually return blob from data available in storage. I didn't really want to update those.
However, I removed this function and added a simple one liner to get the blob from summary.